packager/datapackage_descriptor.json

```
@@ -86,243 +95,232 @@
+                        "required": true,
+                        "minimum": 1,
+                        "maximum": 12
+                    }
                 },
                 {
-                    "name": "funds_origin_pos",
-                    "title": "Origin Position",
-                    "type": "number",
-                    "description": "Benefit received by the employee, because he works in another position of the original.",
-                    "description-ptbr": "Montante recebido de outro órgão, transferido para o empregado"
+                    "name": "ano",
+                    "type": "integer",
+                    "title": "Ano",
+                    "description": "Year of data collection",
+                    "description-ptbr": "O ano que os dados foram coletados",
+                    "constraints": {
+                        "required": true,
+                        "minimum": 2010
+                    }
                 },
                 {
-                    "name": "funds_others_total",
-                    "title": "Sum of other funds",
-                    "type": "number",
-                    "description": "Others types of benefits that are not described before",
-                    "description-ptbr": "Qualquer outro tipo de benefício não explicitado em outros campos"
+                    "name": "id_funcionario",
+                    "title": "Número do id dentro do pacote de dados",
+                    "type": "string",
+                    "format": "default",
+                    "description": "Employee registration word",
+                    "description-ptbr": "A palavra que refere-se ao identificador de registro do empregado dentro do pacote de dados"
                 },
                 {
-                    "name": "discounts_total",
-                    "title": "Total discounts",
-                    "type": "number",
-                    "description": "Represents the sum of discounts applied to the employee.",
-                    "description-ptbr": "Total de descontos aplicados ao empregado",
+                    "name": "nome_funcionario",
+                    "title": "Nome do Funcionário",
+                    "type": "string",
+                    "description": "Employee Name",
+                    "format": "default",
+                    "description-ptbr": "O nome do empregado",
                     "constraints": {
                         "required": true,
-                        "minimum": "0"
+                        "minLength": 2
                     }
                 },
+
                 {
-                    "name": "discounts_prev_contribution",
-                    "title": "Social security discounts",
-                    "type": "number",
-                    "description": "Total of Social security discounts",
-                    "description-ptbr": "Desconto da contribuição para a previdência social",
+                    "name": "tipo",
```

Para diferenciar do tipo no outro recurso.

```suggestion
                    "name": "tipo_detalhe",
```